.github/workflows/main.yml

```diff
@@ -0,0 +1,62 @@
+name: Wildfly s2i Image CI
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    paths-ignore:
+      - 'docs/**'
+env:
+  LANG: en_US.UTF-8
+  S2I_URI: https://api.github.com/repos/openshift/source-to-image/releases/latest
+  CEKIT_VERSION: 3.2.1
+  CI: False
+jobs:
+  ci:
+    name: Wildfly-s2i Build and Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: n1hility/cancel-previous-runs@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Setup required system packages
+        run: |
+          echo CI: ${CI}
+          echo env.CI: ${{ env.CI }}
+          sudo apt-get update
+          sudo apt-get install krb5-multidev
+      - name: Setup virtualenv and install cekit and required packages
+        run: |
+          sudo pip install virtualenv
+          mkdir ~/cekit${{ env.CEKIT_VERSION }}
+          virtualenv ~/cekit${{ env.CEKIT_VERSION }}
+          . ~/cekit${{ env.CEKIT_VERSION }}/bin/activate
+          pip install cekit==${{ env.CEKIT_VERSION }} docker docker-squash odcs behave lxml
+      - name: install s2i binary
+        run: |
+          echo ===== Installing s2i from ${{ env.S2I_URL }}
+          mkdir /tmp/s2i/ && cd /tmp/s2i/
+          curl -s ${{ env.S2I_URI }} \
+           | grep browser_download_url \
+           | grep linux-amd64 \
+           | cut -d '"' -f 4 \
+           | wget -qi -
+           tar xvf source-to-image*.gz
+           sudo mv s2i /usr/bin
+           which s2i
+           s2i version
+      - name: Build 
+        run: |
+          export CI=${{ env.CI }}
+          . ~/cekit${{ env.CEKIT_VERSION }}/bin/activate
+          make 
+      - name: Test
+        run: |
+          export CI=${{ env.CI }}
+          . ~/cekit${{ env.CEKIT_VERSION }}/bin/activate
+          pushd wildfly-builder-image
+          cekit -v test behave --steps-url https://github.com/luck3y/behave-test-steps.git
```

Comment:
Note this is due to behave-test-steps having an env check on the variable CI, which appears to be set in github actions, I'll open a PR there to remove it.

Reply:
@luck3y , that is very nice! Would it be possible to integrate the test/run script that execute additional tests not run with cekit? These tests are called when "make test" is called.

Reply:
@jfdenise Yeah, we can do that once the test steps issue is fixed. Also, I think it would be better if we make sure that build and test are separated, so we can build and maybe push the image to a test repo in quay. If we do that, then we could speed up rerunning tests, and also have a way for people to pull test images for PRs.

Reply:
+1!